java.java

```diff
@@ -0,0 +1,11 @@
+// pacage descrition
```

Comment:
The file has increased in 11 lines.
<!-- lookout comment separator -->
---
This line exceeded 13 chars.